app/src/main/java/com/example/taskyy/ui/objects/Reminder.kt

```diff
@@ -13,4 +16,31 @@ class Reminder(
     agendaAction: AgendaItemAction
 ) : AgendaEventItem(title, description, alarmType, id, timeInMillis, agendaItem, agendaAction) {
 
+    override fun toReminderEntity(): ReminderEntity {
+        return ReminderEntity(
+            id = eventId,
+            description = description,
+            title = title,
+            remindAt = alarmType,
+            time = timeInMillis,
+            action = agendaAction
+        )
+    }
+
+    fun toReminderDto(): ReminderDTO {
+        return ReminderDTO(
+            id = eventId,
+            description = description,
+            title = title,
+            time = timeInMillis,
+            remindAt = alarmType
+        )
+    }
```

Comment:
Not the place to put mappers. Here you're coupling the entities to your UI layer apparently (at least if we look at the package name). The agenda item model is a project-wide domain model and the mappers for that is data logic, so it should be in your app's data layer